### PR TITLE
Rename bootstrappers and packages to dnx-prefixed

### PR DESCRIPTION
**Pull request by ChengTian**
parent https://github.com/aspnet/Coherence-Signed/issues/19 https://github.com/aspnet/Coherence-Signed/issues/26

Did local `build verify` verification.

@Eilon @muratg @anurse @davidfowl @troydai @NTaylorMullen

### CODE REVIEW

File: nuspec/dnx-clr-win-x64.nuspec [hunk 1]
```diff
@@ -5,7 +5,7 @@
         <version>0.0.1-pre</version>
         <authors>Microsoft</authors>
         <requireLicenseAcceptance>false</requireLicenseAcceptance>
-        <description>The .NET Runtime Environment for Desktop CLR on Windows x64</description>
+        <description>Microsoft .NET Execution environment for Desktop CLR on Windows x64</description>
```
Comment (muratg) [inline nuspec/dnx-clr-win-x64.nuspec]: I assume this change (and others below) are based on a previous PR.

Comment (ChengTian) [inline nuspec/dnx-clr-win-x64.nuspec]: When last time we change .NET back to KRE, we decided to leave this part because it doesn't have any impact and we knew there would be another renaming.

File: scripts/klr.sh [hunk 2]
```diff
@@ -9,7 +9,7 @@ done
 DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
```
Comment (muratg) [inline scripts/klr.sh]: Do we need to rename klr.sh file?

Comment (ChengTian) [inline scripts/klr.sh]: Ah, yes, fixing

File: src/dnx.coreclr/tpa.cpp [hunk 3]
```diff
@@ -49,10 +49,10 @@ BOOL CreateTpaBase(LPWSTR** ppNames, size_t* pcNames, bool bNative)
     }
     else
     {
-        pArray[0] = _wcsdup(L"Internal.Runtime.Uri.dll");
-        pArray[1] = _wcsdup(L"Internal.Uri.dll");
-        pArray[2] = _wcsdup(L"kre.coreclr.managed.dll");
-        pArray[3] = _wcsdup(L"kre.host.dll");
```
Comment (troydai) [inline src/dnx.coreclr/tpa.cpp]: Just to be sure the new copy of this file is still automatically generated, right?

File: src/Microsoft.Framework.PackageManager/project.json [hunk 4]
```diff
@@ -43,9 +43,9 @@
...
-            "%project:Directory%/../../build/batchcopy %project:BuildOutputDir%/Debug/aspnet50/*.* %project:Directory%/../../artifacts/build/kre-mono/bin/lib/Microsoft.Framework.PackageManager",
-            "%project:Directory%/../../build/batchcopy %project:BuildOutputDir%/Debug/aspnetcore50/*.* %project:Directory%/../../artifacts/build/kre-coreclr-win-x86/bin/lib/Microsoft.Framework.PackageManager"
+            "%project:Directory%/../../build/batchcopy %project:BuildOutputDir%/Debug/aspnet50/*.* %project:Directory%/../../artifacts/build/dnx-clr-win-x86/bin/lib/Microsoft.Framework.PackageManager",
+            "%project:Directory%/../../build/batchcopy %project:BuildOutputDir%/Debug/aspnet50/*.* %project:Directory%/../../artifacts/build/dnx-mono/bin/lib/Microsoft.Framework.PackageManager",
+            "%project:Directory%/../../build/batchcopy %project:BuildOutputDir%/Debug/aspnetcore50/*.* %project:Directory%/../../artifacts/build/dnx-coreclr-win-x86/bin/lib/Microsoft.Framework.PackageManager"
         ]
```
Comment (muratg) [inline src/Microsoft.Framework.PackageManager/project.json]: Should aspnet50/aspnetcore50's in folder names reflect the new TFMs?  /cc @NTaylorMullen

File: scripts/klr.sh (hunk 2 above)
Comment (ChengTian) [inline scripts/klr.sh]: Fixed. The bootstrapper for mono is special, so I left it out. All other file names containing 'klr' should already be renamed.

File: src/dnx.clr/dnx.clr.vcxproj [hunk 5]
```diff
@@ -21,8 +21,8 @@
   <PropertyGroup Label="Globals">
     <ProjectGuid>{0E4CB542-5E58-4D24-9CAE-DAC83D932DBE}</ProjectGuid>
     <Keyword>Win32Proj</Keyword>
-    <RootNamespace>krenet45</RootNamespace>
-    <ProjectName>kre.clr</ProjectName>
+    <RootNamespace>dnxnet45</RootNamespace>
```
Comment (anurse) [inline src/dnx.clr/dnx.clr.vcxproj]: I realize RootNamespace is mostly irrelevant in a C++ project, but we should correct this value while we're here.

File: src/dnx.coreclr/tpa.cpp (hunk 3 above)
Comment (ChengTian) [inline src/dnx.coreclr/tpa.cpp]: Yes, this is the one I got after doing `build verify`

File: src/Microsoft.Framework.PackageManager/project.json (hunk 4 above)
Comment (muratg) [inline src/Microsoft.Framework.PackageManager/project.json]: Just chatted with @ChengTian.  The folder renames will be in another PR, as Wei wants to keep things working with this PR.

:+1:

Comment (anurse) [inline src/Microsoft.Framework.PackageManager/project.json]: Not yet, the runtime doesn't understand them or build them. That will happen in an upcoming PR, let's get this one in first

File: src/dnx.coreclr/tpa.cpp (hunk 3 above)
Comment (troydai) [inline src/dnx.coreclr/tpa.cpp]: :+1:

**Comment by anurse** [comment]
Looks good aside from my minor notes. I say :shipit: ASAP and let's find out what breaks :)

**Comment by muratg** [comment]
:shipit:

File: src/dnx.clr/dnx.clr.vcxproj (hunk 5 above)
Comment (ChengTian) [inline src/dnx.clr/dnx.clr.vcxproj]: Good point, fixed
